website/docs/r/cosmosdb_account.html.markdown

```diff
@@ -168,9 +168,7 @@ The following arguments are supported:
 
 * `name` - (Required) The capability to enable - Possible values are `AllowSelfServeUpgradeToMongo36`, `DisableRateLimitingResponses`, `EnableAggregationPipeline`, `EnableCassandra`, `EnableGremlin`, `EnableMongo`, `EnableTable`, `EnableServerless`, `MongoDBv3.4` and `mongoEnableDocLevelTTL`. 
 
-**NOTE:**  Setting `MongoDBv3.4` also requires setting `EnableMongo`.
-
----
+**NOTE:** Setting `MongoDBv3.4` also requires setting `EnableMongo`. Only `DisableRateLimitingResponses` can be removed from an existing Cosmos DB account. Only `AllowSelfServeUpgradeToMongo36`, `DisableRateLimitingResponses`, `EnableAggregationPipeline`,`MongoDBv3.4` and `mongoEnableDocLevelTTL` can be added to an existing Cosmos DB account.
```

Comment:
Capabilities is a free formed string with KeyValuePair semantics. CosmosDB team will add new capabilities as we release new preview features and some of these capabilities can be disabled.  Instead of listing out all capabilities here, would it help if we have a doc's page with a list of all active capabilities and if they are removeable?

Reply:
@sivethe Thanks for your advice here. Re "_Instead of listing all functions here, would it help if we had a documentation page with a list of all active functions and if they were deletable?_", does this mean that terraform provider should have a public documentation to list all active capabilities and if they are removeable? If yes, this file [cosmosdb_account.html.markdown](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cosmosdb_account#capabilities) which I updated here is exactly the documentation on the `capabilities` usage of the resource `azurerm_cosmosdb_account`. However, if CosmosDB team has any official and keep-updated documentation that could better replace the existing explanation, please feel free let me know and I can put the link in this documentation.